lib/src/services/jira.ts

```diff
@@ -13,140 +19,185 @@ export const PaginatedResults = T.readonly(
 );
 
 export const Author = T.type({
-  name: T.string,
-});
-
-export const IssueComment = T.type({
-  id: T.string,
-  author: Author,
-  body: T.string,
-  created: readOnlyDateFromISOString,
-  updated: readOnlyDateFromISOString,
-});
-
-export const ChangeLog = T.type({
-  id: T.string,
-  author: Author,
-  created: readOnlyDateFromISOString,
-  items: T.readonlyArray(
-    T.type({
-      field: T.string,
-      fieldtype: T.string,
-      from: T.union([T.string, T.null]),
-      fromString: T.union([T.string, T.null]),
-      to: T.union([T.string, T.null]),
-      toString: T.union([T.string, T.null]),
-    })
-  ),
+  name: nullOrMissingToUndefined(T.string),
 });
 
-export const IssueWorklog = T.type({
-  author: Author,
-  started: readOnlyDateFromISOString,
-  timeSpentSeconds: T.number,
-  comment: nullOrMissingToUndefined(T.string),
-});
+export const IssueCommentGeneric = (convertISO = true) =>
```

Comment:
Should be no types with a `Generic` suffix.

Reply:
Should be no boolean flags. As discussed, pass in the code to be used.

Reply:
corrected